function_calling/src/tools.py

```diff
@@ -0,0 +1,163 @@
+import os
```

Comment:
I would add to the readme a link to this file of examples if you haven't already. If you already did, please ignore.